### ISSUE TITLE
DDR flash failure?

### ISSUE THREAD

**Issue by alec-hewitt** [OpenNuvoton/NUC980_NuWriter_CMD#15]
Hello there. I've developed a custom board with the NUC980DK61YC based on the IIOT evaluation board. I am having trouble programming the chip. It looks like it is able to see the device, but not flash the internal DDR. I posted a schematic, is there any reason why this won't work? Programming through USB0.

Thank you

The error:

```
Number of devices: 1
Burn device 1 ...
init_xusb: Burn DDR to Device failed,bResult=-1
initail xusb failed -1
Failed
```
[NUC980_schem.pdf](https://github.com/OpenNuvoton/NUC980_NuWriter_CMD/files/7406995/NUC980_schem.pdf)

Edit: I'll add that the build and NuWriter setup is OK, and I'm able to flash & boot the IIOT Evaluation board.

**Comment by alec-hewitt** [comment]
Hi there,

Thank you for your responses.

I have built a NEW board with the DK61YC and I now have NOR Flash installed. Is there a default configuration that I should use to build my images and u-boot?

I have tried the nuc980_iot_defconfig and nuc980_defconfig. I CAN flash the chip and it DOES verify the image once flashed, but I can't access the console (I.E. it does not appear to be booting correctly).

Also, any recommendations for env.txt boot commands? Mine is currently:

baudrate=115200 bootcmd=sf probe 0 75000000; sf read 0x7FC0 0x200000 0x800000; bootm 0x7FC0 bootdelay=3 stderr=serial stdin=serial stdout=serial

Using the default settings for NOR Flash in run.ini.

Any help appreciated.
